eng/Tools.props

```diff
@@ -6,6 +6,7 @@
       $(RestoreSources)
     </RestoreSources>
     <MSBuildTreatWarningsAsErrors>false</MSBuildTreatWarningsAsErrors>
+    <Language>C#</Language>
```

Comment:
I know why the change is necessary as I also had to do the same in the xunit depproj but does that mean that we don't support VB analyzers?

Reply:
IIRC we looked at this and found that at the moment all of our analyzers happen to be language agnostic.  @safern and I did discuss this and if we wanted to support both we could re-resolve analyzers from the assets file and generate two different props, but at the moment didn't see the need because a) buildtools didn't do it, and b) none of our analyzers would differ.  Let us know if this would be different for Xunit